web/docs/tutorials/dev-excuses-app/perform-migration-and-run.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Perform migration and run the app
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Before starting an app – we need to execute database migration because we changed the DB schema by adding new entities. If you’ve had something running in the terminal – stop it and run:
```

Comment:
Maybe we can explain this a bit more / simpler?
Smth like "By doing migration, we sync the database schema with the entities we defined". Hm.